chrome/content/zotero/xpcom/plugins.js

```diff
@@ -57,22 +60,63 @@ Zotero.Plugins = new function () {
 			registerLocales(addon);
 			await _callMethod(addon, 'startup', REASONS.APP_STARTUP);
 		}
-		
+
 		Zotero.addShutdownListener(async () => {
 			var { addons } = await AddonManager.getActiveAddons(["extension"]);
 			for (let addon of addons) {
 				await _callMethod(addon, 'shutdown', REASONS.APP_SHUTDOWN);
 			}
 		});
+
+		const mainWindowListener = {
+			onOpenWindow: function (aWindow) {
+				const domWindow = aWindow
+					.QueryInterface(Ci.nsIInterfaceRequestor)
+					.getInterface(Ci.nsIDOMWindowInternal || Ci.nsIDOMWindow);
```

Comment:
→ `let domWindow = xulWindow.docShell.domWindow;` (and below as well)

(Also, we generally avoid `const` in Zotero code, except occasionally for explicit scalar constants.)

Reply:
Done